run-make-check.sh

```diff
@@ -60,45 +76,63 @@ function run() {
         echo "This probably means distribution $ID is not supported by run-make-check.sh" >&2
     fi
 
+    if ! type ccache > /dev/null 2>&1 ; then
+        echo "ERROR: ccache could not be installed"
+        exit 1
+    fi
+    if ! type jq > /dev/null 2>&1 ; then
```

Comment:
no need to install `jq`. it is installed by `mk-build-deps`.

Reply:
@tchaikov If the user has a local ccache configuration file, the script saves it before changing the local ccache config, and then restores it upon exit. If we do not trap ERR, the saved file does not get restored.